### PR TITLE
Add Python 3.7 to test workflow

### PR DESCRIPTION
**Pull request by vtomole**

### CODE REVIEW

**Comment by ncrubin** [comment]
why is 3.7 just for windows and mac?  My takeaway from the discussion last week was that we wanted 3.7 test along with 3.6 testing.  We can do the 3.7 testing only on the ubuntu build.

**Comment by vtomole** [comment]
@ncrubin  Just added a check for 3.7

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -97,6 +97,22 @@ jobs:
           git config --global user.name ${GITHUB_ACTOR}
       - name: Pytest check
         run: check/pytest --actually-quiet
+  pytest37:
+    name: Pytest Ubuntu
```
Comment (kevinsung) [inline .github/workflows/ci.yml:101]: Update the name of this and the 3.6 check so that they are differentiated from each other.

Comment (vtomole) [reply .github/workflows/ci.yml]: Done.

Comment (kevinsung) [reply .github/workflows/ci.yml]: Let's be a little more explicit. Call this check "Pytest Ubuntu Python 3.7" and the other check "Pytest Ubuntu Python 3.6"

Comment (vtomole) [reply .github/workflows/ci.yml]: Done.

**Comment by ncrubin** [comment]
@kevinsung all checks have passed.  Not sure why it is waiting for the old pytest ubuntu run.

**Comment by vtomole** [comment]
@ncrubin It's cause the repo is set for that test to pass before the merge can be done. It needs to be turned off.

**Comment by ncrubin** [comment]
Thanks @vtomole .  I've updated the test requirements for the repo.  pytest ubunutu is removed and pytest ubuntu 3.6 & 3.7 have been added.

**Pull request merged**